### PR TITLE
Add events view and order placing

### PR DESCRIPTION
**Pull request by Ykm22**

### CODE REVIEW

File: TicketManagementSystem/src/scripts/handlers/handleTicketPurchase.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { fetchSaveOrder } from "../network/saveOrder";
+export async function handleTicketPurchase(ticketCategoryId, numberOfTickets, eventId, api_urls){
+    // manually entered userId
+    let userId = '8A029008-1AAF-46E3-B319-0006D111F229';
```
Comment (endv-bogdanb) [inline TicketManagementSystem/src/scripts/handlers/handleTicketPurchase.js:4]: If you don't change the value, you can use const.
```suggestion
    const userId = '8A029008-1AAF-46E3-B319-0006D111F229';
```

File: TicketManagementSystem/src/scripts/network/getEvents.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export async function fetchEvents(url){
+    console.log(url + 'events');
+    let response = await fetch(url + 'events',
```
Comment (endv-bogdanb) [inline TicketManagementSystem/src/scripts/network/getEvents.js:3]:
```suggestion
    let response = await fetch(`${url}events`,
```

File: TicketManagementSystem/src/scripts/utils/getDates.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export function getDates(event){
```
Comment (endv-bogdanb) [inline TicketManagementSystem/src/scripts/utils/getDates.js:1]: Can you use JS date to parse it ?

**Comment by endv-bogdanb** [comment]
Please add a video / screenshot with the implementation

**Comment by Ykm22** [comment]
> Please add a video / screenshot with the implementation

Added demo_v1 in _Teams/Frontend/Ștefan Ichim/demo_

**Comment by endv-bogdanb** [comment]
> > Please add a video / screenshot with the implementation
> 
> Added demo_v1 in _Teams/Frontend/Ștefan Ichim/demo_

you can upload it in description if is under 10mb.

**Comment by Ykm22** [comment]
> > > Please add a video / screenshot with the implementation
> > 
> > 
> > Added demo_v1 in _Teams/Frontend/Ștefan Ichim/demo_
> 
> you can upload it in description if is under 10mb.

image
😞

**Review by endv-bogdanb — APPROVED** [review]

**Pull request merged**